.travis.yml

```diff
@@ -39,7 +46,13 @@ matrix:
 before_script:
     - sudo ./scripts/configure_postgres.sh
 
-script: $SCRIPT
+script: 
+    - if [[ "${TRAVIS_CPU_ARCH}" == "arm64" ]]; then
+        sudo chmod +x $SCRIPT;
+        travis_wait 40 $SCRIPT;
```

Comment:
Is this still required here? Does it mean that the build script is expected to not emit any output for close to 40 minutes?

Reply:
It takes more than 30 mins to build and in travis it throws an error if there is no output for 10 mins. It will emit output immediately if any error. If no output then it is running without any issues.

Reply:
Yes, I have noticed that [the build takes 34 mins](https://travis-ci.org/github/odidev/psycopg2-wheels/builds/709916473). Is that expected?

Reply:
Yes, it takes more than 30 mins to build in aarch64.

Reply:
Removed `travis_wait 40`, after adding architecture check in perl installation. It has passed in my local [build](https://travis-ci.org/github/odidev/psycopg2-wheels/builds/710054368).